ush/templates/FV3LAM_wflow.xml

```diff
@@ -614,10 +621,51 @@ MODULES_RUN_TASK_FP script.
 <!--
 ************************************************************************
 ************************************************************************
+-->
+  <task name="&RUN_FCST_TN;{{ uscore_ensmem_name }}" cycledefs="forecast_12z" maxtries="{{ maxtries_run_fcst }}">
+
+    &RSRV_FCST;
+  {% if run_realtime %}
+    &WALL_LIMIT_FCST_12Z;
+  {% endif %}
+
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_FCST_TN;" "&JOBSDIR;/JREGIONAL_RUN_FCST"</command>
+  {% if machine in ["JET", "HERA"]  %}
+    <cores>{{ ncores_run_fcst }}</cores>
+    <native>{{ native_run_fcst }}</native>
+  {% else %}
+    <nodes>{{ nnodes_run_fcst }}:ppn={{ ppn_run_fcst }}</nodes>
+    <nodesize>&NCORES_PER_NODE;</nodesize>
+  {% endif %}
+    <walltime>{{ wtime_run_fcst }}</walltime>
+    <jobname>rrfse_&RUN_FCST_TN;{{ uscore_ensmem_name }}</jobname>
+    <join><cyclestr>&LOGDIR;/&RUN_FCST_TN;{{ uscore_ensmem_name }}_@Y@m@d@H.log</cyclestr></join>
+
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+    <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr>{{ slash_ensmem_subdir }}</cyclestr></value></envar>
+    <envar><name>ENSMEM_INDX</name><value><cyclestr>#{{ ensmem_indx_name }}#</cyclestr></value></envar>
+  
+    <dependency>
+      <and>
+    {%- if run_realtime %}
+        <timedep><cyclestr offset="&START_FCST_12Z;">@Y@m@d@H@M00</cyclestr></timedep>
+    {%- endif %}
+        <taskdep task="&MAKE_ICS_TN;{{ uscore_ensmem_name }}"/>
+        <taskdep task="&MAKE_LBCS_TN;{{ uscore_ensmem_name }}"/>
+      </and>
+    </dependency>
+
+  </task>
+<!--
+************************************************************************
+************************************************************************
 -->
   <metatask name="&RUN_POST_TN;{{ uscore_ensmem_name }}">
     <var name="fhr"> {% for h in range(0, fcst_len_hrs+1) %}{{ " %03d" % h  }}{% endfor %} </var>
```

Comment:
Should `fcst_len_hrs` be `ext_fcst_len_hrs`?

Reply:
It is `fcst_len_hrs` for post jobs. The variable `ext_fcst_len_hrs` is only used for looking for the last LBC file. It is actually ${FCST_LEN_HRS}+${EXTRN_MDL_LBCS_OFFSET_HRS}, when `EXTRN_MDL_LBCS_OFFSET_HRS` is greater than 0. For example, it is  6 for GEFS, and therefore the last GEFS forecast file is hour 66 when the forecast length is 60.
Thank you for reviewing it!